output/csv/config.go

```diff
@@ -92,7 +99,8 @@ func ParseArg(arg string) (Config, error) {
 
 // GetConsolidatedConfig combines {default config values + JSON config +
 // environment vars + arg config values}, and returns the final result.
-func GetConsolidatedConfig(jsonRawConf json.RawMessage, env map[string]string, arg string) (Config, error) {
+//nolint: lll
```

Comment:
Passing the additional logger parameter causes the line length linting error, but given we're only passing the logger until people have migrated to the camel case version I felt the no lint comment would be sufficient for the time being.

Reply:
I in general prefer to split the `func <name>(<arguments>) (<results>){` on three lines as in 
```
func <name>(
    <arguments>,
)(<results>){ 
```
as it keeps it more readable especially when lines start to wrap.